homeassistant/components/netgear/sensor.py

```diff
@@ -432,7 +432,7 @@ async def async_added_to_hass(self) -> None:
             if sensor_data is not None:
                 self._value = sensor_data.native_value
             else:
-                self.schedule_update_ha_state()
+                self.coordinator.async_request_refresh()
```

Comment:
This is a coroutine function and we should await it.

Reply:
Fix is here: https://github.com/home-assistant/core/pull/85574
My appologies, I schould have checked this better, now I booted up my dev enviroment to ensure it is now working.